Dockerfile.template

```diff
@@ -21,13 +21,13 @@ RUN	rm -rf /var/lib/mysql && \
 		| xargs -0 grep -lZE '^(bind-address|log|user\s)' \
 		| xargs -rt -0 sed -Ei 's/^(bind-address|log|user\s)/#&/' || true
 
-RUN mkdir /docker-entrypoint-initdb.d
```

Comment:
This doesn't seem necessary. I don't see any reason to keep it.

I realize this is also in the PostgreSQL Dockerfile, which if we remove it here, we should likely remove it there as well.